config/infection.json.dist

```diff
@@ -0,0 +1,26 @@
+{
+    "source": {
+        "directories": [
+            "src/Psl/"
+        ],
+        "excludes": [
+            "RandomSequence",
+            "Async",
+            "IO",
+            "Unix",
+            "TCP",
+            "Internal",
+            "{.*/Internal/.*}"
```

Comment:
most async IO stuff, result either in timeouts, or false-positives, e.g:

change `timeout` in `stream_socket_accept($resource, timeout: 0.0)` from `0.0` to `1.0`, but this is never called unless the server socket is ready, so if it happened to go above the timeout ( 0.0 ), that's a bug in PHP, not PSL ( which we handle by throwing a runtime exception, but exclude it from code coverage, as it can't be tested ), and by increasing the timeout, nothing happens, as it returns the socket immediately

Reply:
maybe if we could enable some mutators for these files only, it would make sense to include them, however, we want all default mutators to be enabled for the rest of the project.